js/logos.json

```
@@ -438,4 +438,16 @@
 	"title": "Whogohost",
 	"filename": "whogohost",
 	"category": ["Web Host", "Domain Registrar"]
+}, {
+	"title": "TechCabal",
+	"filename": "techcabal",
+	"category": ["Blog"]
```

```suggestion
	"category": ["Media"]
```

Hey, I think this a more appropriate categorization for TC.

---

@fathermerry I agree. Fixed, thanks.